recognition/include/pcl/recognition/mask_map.h

```diff
@@ -65,10 +65,16 @@ namespace pcl
       inline const unsigned char* 
       getData () const { return (&data_[0]); }
 
+      [[deprecated("output parameter is not desirable,\
```

Comment:
```suggestion
      [[deprecated("use getDifferenceMask() with return value instead")]]
```

For deprecation messages, we generally try to stick to "use _%signature%_ instead" template. The _%signature%_ part does not have to be a formal signature but should be sufficient to understand which function/overload is meant.

Reply:
Ok, cheers